3.3/examples/django/README.md

```diff
@@ -0,0 +1,91 @@
+Django application example
+==========================
+
+This is a basic [Django](https://www.djangoproject.com/) project and some helper scripts to demonstrate how to deploy Python web applications with [OpenShift 3](http://www.openshift.org).
+
+We will be using Django 1.8 and the [PostgreSQL](http://www.postgresql.org/) database.
+
+OpenShift supports multiple deployment strategies. This example uses the Source-To-Image strategy, and the [Python 3.3 STI image](https://github.com/openshift/sti-python/tree/master/3.3) for producing application builds.
+
+**WARNING**: this is a _proof of concept_ example and, therefore, the code available here is not meant to be used in production.
+
+
+Application Build, Deploy, and Update Flow
+------------------------------------------
+
+Start by cloning this repository and changing your current directory appropriately:
+
+```bash
+git clone https://github.com/openshift/sti-python.git
+cd sti-python/3.3/examples/django
+```
+
+Follow the instructions in https://github.com/openshift/origin/tree/master/examples/sample-app to build the `openshift` all-in-one binary and start your OpenShift 3 server and docker registry. You won't need to clone that repository, and you should use `sti-python/3.3/examples/django` as your working directory.
```

Comment:
It's very hard to give pointers to a quick-changing document. I saw in other README's references like "follow steps 1-9", and those numbers are now outdated. That full line is mentioned right in the next section, and IMHO it is long and daunting enough to appear twice in a row.
I opted for briefly mentioning the main things you need to have ready -- in the next line below